src/array_practice.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+int main()
+{
+	int numbers[] = {1, 4, 5, 6, 10};
+	char *strings[] = {"Hello", "World", "or", "Hello", "Sasha"};
+	int x;
+	int sum = 0;
+
+	printf("Enter number x for multiply an array : ");
+	scanf("%d", &x);
+
+	for(int i = 0; i < sizeof(numbers) / sizeof(numbers[0]); i++) {
```

Comment:
Follow the requirements:
1. Create a function, that will receive an array of 5 integers, sum them and return a result.
2. Create a function, that will receive an array of 5 integers, multiply them by x (input parameter) and return them.